deploy/clusterrole.yaml

```
@@ -7,10 +7,6 @@ rules:
   - ""
   resources:
   - nodes
-  - nodes/status
-  - nodes/metrics
-  - nodes/spec
-  - nodes/stats
   verbs:
   - get
   - list
```

```suggestion
  - list
  - watch
```
Looks like deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml contains correct verbs, but clusterrole.yaml not

---

added `watch`... let's see if we are converging :)